pyart/testing/__init__.py

```diff
@@ -46,5 +47,6 @@
 from .sample_objects import make_empty_rhi_radar
 from .sample_objects import make_velocity_aliased_rhi_radar
 from .tmpdirs import InTemporaryDirectory
+from .sample_objects import make_target_grid, make_storm_grid, make_normal_storm
```

Comment:
my bad.. will fix
Will also remove the conditional import


> On Aug 9, 2016, at 10:45 AM, Jonathan J. Helmus <notifications@github.com> wrote:
> 
> In pyart/testing/__init__.py <https://github.com/ARM-DOE/pyart/pull/565#discussion_r74085099>:
> 
> > @@ -46,5 +47,6 @@
> >  from .sample_objects import make_empty_rhi_radar
> >  from .sample_objects import make_velocity_aliased_rhi_radar
> >  from .tmpdirs import InTemporaryDirectory
> > +from .sample_objects import make_target_grid, make_storm_grid, make_normal_storm
> Only make_storm_grid should be imported on this line, the other two function are already imported earlier
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/ARM-DOE/pyart/pull/565/files/6a1724c8fca6d2a7f7c7175ceb51684ba82a0b95#r74085099>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAyYB4IIBb65z1SayHaRU4RIVoA4hGTvks5qeKC0gaJpZM4JgLvz>.
>